teams/teams-ps/teams/Set-CsTeamsMeetingPolicy.md

```diff
@@ -1369,6 +1371,29 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -LobbyChats
+
+NOTE: This feature has not been released yet and will have no changes if it is enabled or disabled.
```

Comment:
Please update to
```
>[!NOTE]
>This feature has not been released yet and will have no changes if it is enabled or disabled.
```

It will look similar to this on the webpage:
<img width="584" alt="image" src="https://github.com/user-attachments/assets/f28f04bd-a920-49ea-870b-fae5c65cd78b">